### ISSUE TITLE
Flake8 rule 'W291' reported an invalid point

### ISSUE THREAD

**Issue by jmason86** [AtomLinter/linter-flake8#570]
Flake8 reported an invalid point for the rule `W291`, with the messge `trailing whitespace`.

```# -*- coding: utf-8 -*-```

at the very top of a file.

Debug information:
Atom version: 1.23.3
Flake8 version: `3.4.1 (mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.5.0) CPython 2.7.13 on Darwin`

**Comment by ddaanet** [comment]
Please save the the file that caused the issue and report it to the flake8 issue tracker.

https://gitlab.com/pycqa/flake8/issues

**Comment by jmason86** [comment]
Ah shoot, the error has gone away so I'm not able to save the file with the error. I haven't touched that line of code so I don't know what fixed it. I did install an additional module in Atom (symbols-tree-view) since initially seeing this error. I haven't rebooted my machine or Atom. If I see it again, I'll report it over on gitlab.

**Comment by ddaanet** [comment]
Usually those errors are triggered by something seemingly unrelated in the same file.

They have nothing to do with with other Atom packages: they are errors in linter tool (flake8 here).

Since you can no longer reproduce it, I am closing the issue.
